static/app.css

```diff
@@ -54,3 +58,9 @@ in different components */
     font-size: 0.16in !important;
   }
 }
+
+@media only screen and (max-width: 400px) {
+  html {
+    font-size: 26px;
```

Comment:
perhaps try a smaller size? This seems massive in the menu component.

Reply:
Decided to remove this completely in favor of having only 18px root font size throughout the app